sonar-db/src/main/resources/org/sonar/db/ce/CeQueueMapper.xml

```diff
@@ -29,8 +29,13 @@
     where cq.uuid=#{uuid}
   </select>
 
-  <select id="countByStatus" parameterType="org.sonar.db.ce.CeQueueDto$Status" resultType="int">
-    select count(1) from ce_queue where status=#{status}
+  <select id="countByStatusAndComponentUuid" parameterType="map" resultType="int">
+    select count(1)
+    from ce_queue
+    where status=#{status}
```

Comment:
You should ask the Atlas team if we should not add an index on CE_QUEUE.STATUS, but I really think we should !